lib/db.js

```diff
@@ -244,7 +244,7 @@ Object.defineProperty(Db.prototype, 'slaveOk', {
 Object.defineProperty(Db.prototype, 'readConcern', {
   enumerable: true,
   get: function() {
-    return this.s.readConcern || { level: 'local' };
+    return this.s.readConcern || {};
```

Comment:
At the point of command construction a value of `null` or `undefined` for `readConcern` should be encoded as `{}`, so I think we should just `return this.s.readConcern` here